src/components/Slider/styled/inputRange.js

```diff
@@ -85,35 +95,35 @@ const StyledInputRange = styled.input`
     }
 
     &::-webkit-slider-thumb:hover {
-        background-color: ${COLOR_BRAND_ACTIVE};
+        background-color: ${props => props.successMainColor};
```

Comment:
Check this style since the color use was brand dark and now it is changed to success